src/components/ErrorsContainer.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component, PropTypes } from 'react';
+import FieldConnect from './FieldConnect';
+import ErrorField from './ErrorField';
+
+class ErrorsContainer extends Component {
+    constructor(props) {
```

Comment:
In this case you dont need getErrors.bind and constructor.

Reply:
Removed